packages/samples/test/output/grpc-library-example/openapi.json

```diff
@@ -134,7 +134,7 @@
         ],
         "responses": {
           "204": {
-            "description": "No content"
+            "description": "No body returned"
```

Comment:
Where does this description change comes from?

Reply:
Good question.  I had to hunt it down myself.  It comes from [here](https://github.com/microsoft/cadl/blob/main/packages/samples/grpc-library-example/library.cadl#L266).